apps/monitoring/kube-prometheus-stack/prometheus-values.enc.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
Shouldn't this file should go into the sbox folder?

Reply:
or deleted

Reply:
This is identical to the sbox one, it was there originally before I split them into dev and sbox values. It's referred to in the [base kustomization](https://github.com/hmcts/sds-flux-config/pull/1450/files#diff-65745e18daabba5c572313d156928e768d256eecc9d206ac5ec3cf9b1e43ffc5) as a default I think but not sure if it's necessary

Reply:
yeah I don't think it needs to be there, I'd delete the file and the reference

Reply:
done